### PR TITLE
Adding demo3

### PR DESCRIPTION
**Pull request by bfuzzy1**
Pinged @preludeorg/wizards (Harry) for verbiage review.

### CODE REVIEW

File: shell/demo/demo3.ps1 [hunk 1]
```diff
@@ -0,0 +1,173 @@
...
+$TEST_ID="07d31938-92fa-40aa-b8af-77a54055f481"
+$TEST_INTRO="Will your computer quarantine a malicious executable file?
+
+Malicious files are used to gain entry and conduct cyberattacks against corporate systems through seemingly
+innocuous email attachments or direct downloads. For example - a malicious Windows executable was used by the SteelClover group 
+during an attack in January 2023.
+
+This test will attempt to download a malicious file to your disk - a defanged version of RedLine Stealer used by SteelClover - 
+to see how your machine will respond. Downloading malicious files can open you up to further attacks,
+so the ability to quickly detect and quarantine any potentially harmful files is an important part of maintaining a proper security posture.
```
Comment (harry-prelude) [inline shell/demo/demo3.ps1:24]: This is more of a marketing thing, but i wonder if we want to start owning 'active security posture' to connect more with 'Active Security Intellignce'.

File: shell/demo/demo3.ps1 [hunk 2]
```diff
@@ -0,0 +1,173 @@
...
+
+$TEST_ID="07d31938-92fa-40aa-b8af-77a54055f481"
+$TEST_INTRO="Will your computer quarantine a malicious executable file?
+
+Malicious files are used to gain entry and conduct cyberattacks against corporate systems through seemingly
+innocuous email attachments or direct downloads. For example - a malicious Windows executable was used by the SteelClover group 
+during an attack in January 2023.
+
+This test will attempt to download a malicious file to your disk - a defanged version of RedLine Stealer used by SteelClover - 
+to see how your machine will respond. Downloading malicious files can open you up to further attacks,
```
Comment (harry-prelude) [inline shell/demo/demo3.ps1:23]: do we want to say 'how your machine will respond' or 'how your endpoint defenses will respond'?

File: shell/demo/demo3.ps1 [hunk 3]
```diff
@@ -0,0 +1,173 @@
...
+$PRELUDE_API = FromEnv "PRELUDE_API" "https://api.preludesecurity.com"
+
+$TEST_ID="07d31938-92fa-40aa-b8af-77a54055f481"
+$TEST_INTRO="Will your computer quarantine a malicious executable file?
+
+Malicious files are used to gain entry and conduct cyberattacks against corporate systems through seemingly
+innocuous email attachments or direct downloads. For example - a malicious Windows executable was used by the SteelClover group 
+during an attack in January 2023.
+
+This test will attempt to download a malicious file to your disk - a defanged version of RedLine Stealer used by SteelClover -
```
Comment (harry-prelude) [inline shell/demo/demo3.ps1:22]: Let's see if we can write these and reorder this description so 'defanged' read before 'malicious'.

Not sure if this is accurate, but here's my stab:

This test will attempt to download a defanged malicious file to your disk - which includes a signature for the RedLine Stealer used by SteelClover - to see how your machine will respond.

**Review by harry-prelude — APPROVED** [review]

File: shell/demo/demo3.ps1 (hunk 2 above)
Comment (bfuzzy1) [reply shell/demo/demo3.ps1:23]: ah yes let I'll make that change.

File: shell/demo/demo3.ps1 (hunk 1 above)
Comment (bfuzzy1) [reply shell/demo/demo3.ps1:24]: Active does not equal proper when it comes to security posture.

**Review by harry-prelude — APPROVED** [review]

**Review by privateducky — APPROVED** [review]

**Pull request merged**